vars/family_Debian.yml

```diff
@@ -1 +1,2 @@
-aem_dispatcher_module_path: /usr/lib/apache2/modules
\ No newline at end of file
+aem_dispatcher_module_path: /usr/lib/apache2/modules
```

Comment:
Should we make that "private" as well for consistency?

Reply:
I don't think so. The `_` from `_aem_dispatcher_apache_server_root` is only a indicator that this is internal and to ensure that the os family overwriting property does not have a `_underscore in front`.

Reply:
Yes sure. But is it meant to be consumed and/or set from outside the role? Is there any difference in how the variables are used?
In any case, strictly speaking this is not backward compatible anyway as somebody may already rely on it.

Reply:
I got the point with the backwards compatibility and I will also check this next week.

Reply:
Just to clarify: What *I* am proposing is not backward compatible, so let's just keep it as it is for now. I guess it's 2.x material.